GTG/core/config.py

```diff
@@ -21,6 +21,9 @@
 Classes responsible for handling user configuration
 """
 
+# TODO: Remove toolbar and quick_add options from configuration
+#       They won't be used in GTG 0.4
+
```

Comment:
Can you put this on top of DEFAULTS dict as that's the place that need to be modified?